tchannel/tornado/message_factory.py

```diff
@@ -97,7 +97,7 @@ def build_raw_request_message(self, request, args, is_completed=False):
         if request.state == StreamState.init:
             message = CallRequestMessage(
                 flags=request.flags,
-                ttl=request.ttl * 1000,
+                ttl=request.ttl,
```

Comment:
can we please use units in the variable names for clarity in the internal APIs?

`ttl_ms` and `ttl_secs` is a lot clearer than `ttl`